### PR TITLE
added usage of clean middleware to remove directories outside project root

### LINKED ISSUE #362: Setting options.output to relative directory above project directory causes clean middleware to not work
If you override the `output` directory like so:

```js
module.exports = {
  options: {
    output: '../build'
  },
  use: ['neutrino-preset-react']
};
```

The `clean` middleware/plugin no longer works.

### PR DESCRIPTION
**Pull request by Aditi574**
Fixes #362

### CODE REVIEW

File: docs/packages/clean/README.md [hunk 1]
```diff
@@ -70,6 +70,23 @@ module.exports = {
...
+```js
+
+module.exports = {
+  options: {
+    // override the output directory
+    output: '../build'
+  },
+  // set allowExternal to true 
+  allowExternal: true,
+  use: ['neutrino-preset-react']
```
Comment (eliperelman) [inline docs/packages/clean/README.md]: Could you update these `use` references to our current middleware, `@neutrinojs/react`?

**Comment by Aditi574** [comment]
@eliperelman could you provide the link? I'm unable to find @neutrinojs/react
Thanks

**Comment by Aditi574** [comment]
@eliperelman please review the changes. Thanks!

**Comment by edmorley** [comment]
> please review the changes. Thanks!

Hi! It looks like the issue spotted in the previous review still exists?
https://github.com/mozilla-neutrino/neutrino-dev/pull/723#discussion_r170808602

**Comment by edmorley** [comment]
@Aditi574 do you know if you'll have a chance to update this? If not, I can make the last tweak instead?

**Comment by Aditi574** [comment]
@edmorley I'll fix this in a couple of hours.

**Comment by eliperelman** [comment]
@Aditi574 bump here if you would like to finish this?

**Comment by eliperelman** [comment]
Closing for inactivity.

**Pull request closed without merging**